ui/pages/liquidity.tsx

```diff
@@ -338,7 +338,12 @@ export default function Liquidity() {
                       className="btn btn-primary normal-case font-medium w-full"
                       action={withdrawAndClaimRewards}
                     >
-                      Withdraw all and claim
```

Comment:
We definitely still want the button label

Reply:
Oops, my bad. Didnt mean to delete it